src/views/tables/OrdersTable/OrderActionsCell.tsx

```diff
@@ -33,9 +34,10 @@ export const OrderActionsCell = ({ orderId, status, isDisabled }: ElementProps)
   }, []);
 
   return (
-    <Styled.OrderActions>
-      <Styled.Toolbar>
+    <ActionsTableCell
+      children={[
```

Comment:
instead of using `children` just wrap in `<ActionsTableCell>{children} </ActionsTableCell>`

Reply:
I think that this should still work for checking children.length